Dockerfile

```diff
@@ -71,6 +78,8 @@ RUN cd $CATKIN_WS && \
     catkin build -j2 dbow2_catkin && \ 
     catkin build -j2 g2o_catkin 
 
+RUN xhost +local:docker
```

Comment:
I am not sure, should it be here before using script or not. But it should be in the whole